### ISSUE TITLE
Web component: Content block cards with Video - After closing the dialog visually it is disappeared. But, Screen reader focus is stuck on close button

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#4744]
## Detailed description

On iPhone voiceover

Issue: 
After closing the dialog visually it is disappeared. But, Screen reader focus is stuck on close button. We are unable to navigate using Left/Right swipe. This issue is not found in react Staging.

Expected:
After closing the dialog, User should be able to navigate to the parent page. And the dialog should disappear for the screen reader user as well.

**Comment by RaphaelAmadeu** [comment]
@Praveen-sr 

Can you please check if the issue is still occuring for you? It seems that the focus changes and it's working finelly with chromevox

**Comment by Praveen-sr** [comment]
@RaphaelAmadeu Ram has retested this issue in IOS 13.7 & 14.3 in both Safari and Chrome 87and able to reproduce - Not fixed.

CC @RobertaJHahn

**Comment by RaphaelAmadeu** [comment]
@Praveen-sr 

Could you please try it again using this url?
http://ibmdotcom-web-components-canary.mybluemix.net/?path=/story/components-content-block-cards--with-videos

Thank you!

**Comment by Praveen-sr** [comment]
@RaphaelAmadeu Ram re tested this on the above link, issue not fixed.

**Comment by RaphaelAmadeu** [comment]
https://images.zenhubusercontent.com/193129061/10c1b950-6e1d-4347-92e5-fb24d9015297/video_2021_02_05_17_11_57.mp4

@Praveen-sr 
This is a capture from my Iphone using voice over, isn't this the supposed behavior?

**Comment by Praveen-sr** [comment]
@RaphaelAmadeu Ram re tested the same scenario on iPhone, Screen reader focus is stuck on close button. We are unable to navigate using Left/Right swipe.

**Comment by IgnacioBecerra** [comment]
@Praveen-sr I wasn't able to replicate this behavior using an iPhone 8 on the latest iOS, would you be able to retest, and if possible, provide a video? Thank you!

cc: @RobertaJHahn

**Comment by Praveen-sr** [comment]
@IgnacioBecerra retested in Web component canary contain block card video in VO IOS,
After closing the dialog now the Screen reader focus is not stuck on "close" button. focus is set back on the triggering element. Above issues is working fine now. 

But following regression found, i.e. Even after closing the modal, VO still identifies the player controls at the end of the page which is not visible.
